extensions/oplogPopulator/OplogPopulatorMetrics.js

```diff
@@ -0,0 +1,164 @@
+const { ZenkoMetrics } = require('arsenal').metrics;
+
+const { getStringSizeInBytes } = require('../../lib/util/buffer');
+
+class OplogPopulatorMetrics {
+    /**
+     * @param {Logger} logger logger instance
+     */
+    constructor(logger) {
+        this.acknowledgementLag = null;
+        this.connectorConfiguration = null;
+        this.requestSize = null;
+        this.connectors = null;
+        this.reconfigurationLag = null;
+        this._logger = logger;
+    }
+
+    registerMetrics() {
+        this.acknowledgementLag = ZenkoMetrics.createHistogram({
+            name: 'oplog_populator_acknowledgement_lag_sec',
+            help: 'Delta between a config change in mongo and it getting processed by the oplogPopulator in seconds',
+            labelNames: ['opType'],
+            buckets: [0.001, 0.01, 1, 10, 100, 1000, 10000],
+        });
+        this.connectorConfiguration = ZenkoMetrics.createCounter({
```

Comment:
is it the number of "reconfiguration" (e.g. running the whole loop)?
or the number of time we update the configuration of a single connector?